dotnet/src/Connectors/Connectors.Onnx/OnnxKernelBuilderExtensions.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.IO;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.SemanticKernel.Connectors.Onnx;
+using Microsoft.SemanticKernel.Embeddings;
+
+namespace Microsoft.SemanticKernel;
+
+#pragma warning disable CA2000 // Dispose objects before losing scope
+
+/// <summary>
+/// Provides extension methods for the <see cref="IKernelBuilder"/> class to configure ONNX connectors.
+/// </summary>
+public static class OnnxKernelBuilderExtensions
+{
+    /// <summary>Adds a text embedding generation service using a BERT ONNX model.</summary>
+    /// <param name="builder">The <see cref="IKernelBuilder"/> instance to augment.</param>
+    /// <param name="onnxModelPath">The path to the ONNX model file.</param>
+    /// <param name="vocabPath">The path to the vocab file.</param>
+    /// <param name="options">Options for the configuration of the model and service.</param>
+    /// <param name="serviceId">A local identifier for the given AI service.</param>
+    /// <returns>The same instance as <paramref name="builder"/>.</returns>
+    public static IKernelBuilder AddBertOnnxTextEmbeddingGeneration(
+        this IKernelBuilder builder,
+        string onnxModelPath,
+        string vocabPath,
+        BertOnnxOptions? options,
```

Comment:
options in service collection has default (null) should this also be on the extensions?
```suggestion
        BertOnnxOptions? options = null,
```

Reply:
Yup, should be optional